ros/openapi/openapi.json

```diff
@@ -259,6 +259,26 @@
                                 }
                             }
                         }
+                    },
+                    "404": {
+                        "description": "System doesn't exist",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Message"
+                                }
+                            }
+                        }
+                    },
+                    "400": {
```

Comment:
Should this be 422?

Reply:
Details about 422 already present above.

Reply:
Ah okay, in that case shall we update the desc. L274?

Reply:
L254 suggestion: _Invalid choice of input for rating_, as discussed L274 looks good